Mathlib/Analysis/InnerProductSpace/Adjoint.lean

```diff
@@ -68,7 +68,7 @@ variable [CompleteSpace E] [CompleteSpace G]
 /-- The adjoint, as a continuous conjugate-linear map. This is only meant as an auxiliary
 definition for the main definition `adjoint`, where this is bundled as a conjugate-linear isometric
 equivalence. -/
-def adjointAux : (E →L[𝕜] F) →L⋆[𝕜] F →L[𝕜] E :=
+noncomputable def adjointAux : (E →L[𝕜] F) →L⋆[𝕜] F →L[𝕜] E :=
```

Comment:
Ideally we'd have a tracking issue for "we made X computable due to a maxrss issue"

Reply:
Where do we want it?

Reply:
I think mathlib4 is fine, unless you want to open a Lean4 one. We can always link from the mathlib one to the lean one at a later date.

Reply:
#7103